pymorphy3/opencorpora_dict/storage.py

```diff
@@ -142,7 +143,7 @@ def _dawg_len(dawg):
     ])
 
 
-def load_meta(filename):
+def load_meta(filename) -> Union[collections.OrderedDict, Dict]:
```

Comment:
It's excessive: `collections.OrderedDict` is a subclass of `dict`, so you can just leave `dict` annotation

Reply:
But OrderedDict is a different type, it has a special method `.move_to_end`, so it makes sense to use the actual types in the annotation.

Reply:
@Andrej730 here it just used to have ordered dict (the same key order as they were added).

Reply:
If methods are not used, then I assume it's initializing it as OrderedDict to indicate that this dictionary will preserve the insertion order <Python 3.7, so why not indicate it in the annotation?

Reply:
@Andrej730 if methods of OrderedDict are not used explicitly, then it can be annotated just with the common superclass. It will be cleaner and less confusing.

Reply:
Okay, I disagree but I guess it's okay since most of the users are probably Python 3.7+ anyway.